### PR TITLE
#984 Symfony 4.4 compatibility

### PR DESCRIPTION
**Pull request by ctrl-f5**
Fix fox issue #984 and Symfony 4.4 compatibility

see https://github.com/symfony/symfony/issues/32394

### CODE REVIEW

File: pkg/enqueue/Doctrine/DoctrineConnectionFactoryFactory.php [hunk 1]
```diff
@@ -2,16 +2,16 @@
 
 namespace Enqueue\Doctrine;
 
+use Doctrine\Bundle\DoctrineBundle\Registry;
```
Comment (DamienHarper) [inline pkg/enqueue/Doctrine/DoctrineConnectionFactoryFactory.php]: You should use `Doctrine\Common\Persistence\ManagerRegistry` instead as noted in the [upgrade doc](https://github.com/doctrine/DoctrineBundle/blob/master/UPGRADE-2.0.md#service-aliases)

Comment (ctrl-f5) [reply pkg/enqueue/Doctrine/DoctrineConnectionFactoryFactory.php]: updated to `ManagerRegistry`

**Comment by luca-simonetti** [comment]
any ETA on this to be merged?

**Comment by Steveb-p** [comment]
> any ETA on this to be merged?

As mentioned in #988 it should be soon, probably tomorrow. In the meantime you can use the workaround described there to see if it will work in your project and maybe spot any issues.

**Comment by ctrl-f5** [comment]
I'll close this one since it's been superseded by #988

**Pull request closed without merging**